docs/tutorials/runtime/cpp.rst

```diff
@@ -219,6 +219,7 @@ MLC-compiled models can be integrated into any C++ project using TVM's C/C++ API
         device_id,
       );
       // Step 2. Obtain all available functions in `mlc_llm`
+      PackedFunc embed = mlc_llm->GetFunction("embed");
```

Comment:
You don't need to change this file for now, because it's just a demo showing how to use C++ APIs.

Reply:
Reverted in https://github.com/mlc-ai/mlc-llm/pull/408/commits/ebab0c51c7a682a992f9a00a5e979c1e117d985e.

Reply:
Thank you!